package.json

```diff
@@ -62,11 +62,11 @@
     "lint-check": "tsc --noEmit && eslint \"*/**/*.{js,ts,tsx}\" NOTICE --max-warnings 0",
     "test": "run-s test:unit test:e2e",
     "test:unit": "jest --runInBand --config=./test/unit/jest.config.js",
-    "test:unit:coverage": "jest --runInBand --config=./test/unit/jest.config.js --coverage",
-    "test:unit:watch": "jest --runInBand --config=./test/unit/jest.config.js --coverage --watchAll",
+    "test:unit:coverage": "npm run test:unit -- --coverage",
```

Comment:
:question: did you try `run-s` instead of npm (which probably fork a new process). This applies to all scripts that have been chandeg to update another script.

Reply:
I didn't think to do that ^^

Reply:
I don't think it's adapted to what we want to do.
Here we just want to run the same configuration as for the unit test, but with one more paramter (for the coverage).

I tried with `run-s`. It doesn't work. After the documentation reading of `run-s`, it's more to run mutliple scripts (it's not what we do).

Reply:
So let's keep it